rabbitmq-default-user-credential-updater.yaml

```
@@ -0,0 +1,27 @@
+package:
+  name: rabbitmq-default-user-credential-updater
+  version: 1.0.4
+  epoch: 0
+  description: Updates RabbitMQ default user password
+  copyright:
+    - license: MPL-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/rabbitmq/default-user-credential-updater
+      tag: v${{package.version}}
+      expected-commit: ba4bb24402338245e8cc4fba062900993a0c94a4
+
+  - uses: go/build
+    with:
+      packages: .
+      output: default-user-credential-updater
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: rabbitmq/default-user-credential-updater
+    strip-prefix: v
```

```suggestion
    strip-prefix: v
    
test:
  pipeline:
    - runs: |
        default-user-credential-updater -h  2>&1 | grep "Usage of default-user-credential-updater:"
```

Adding this will run the binary and check for the help string. It isn't much functionally, but it at least shows that the binary runs

---

Good call! Added.